NSS-Day-Cohort-64/rare-django-server-rare-django-server-init-to-win-it
# Kz commentview

Added comment view and post view--
to test: 
git checkout kz-commentview

Open Postman
do GET request to http://localhost:8000/posts with auth token: 06787add96eb3264080015997720cb0877b40f90
You should see a list of posts.  Add /1 at the end to get a single post

Do a POST request to http://localhost:8000/posts
 {
        "author": 1,
        "title": "Logan Fan Club?",
        "category": 1,
        "publication_date": "2023-08-29T15:00:00Z",
        "content": "Who wants to start a fan club for Logan?"
   }

Do a PUT request to http://localhost:8000/posts/(whatever post you created)
    {
        "title": "LOGAN ROCKS",
        "category": 1,
        "publication_date": "2023-08-30T15:00:00Z",
        "content": "LOGAN IS THE BEST"
    }
Do a get request to the same post to make sure it updates

Do a DELETE request to http://localhost:8000/posts/(whatever post you created)
Do a get request to verify that the post was deleted.

Do a GET request to http://localhost:8000/comments
You should see a list of comments.  Add /1 at the end to get a single comment

Do a POST request to http://localhost:8000/comments
    {
        "post": 1,
        "time_stamp": "2023-08-29T15:00:00Z",
        "content": "Wow, this post is so boring"
    }

Do a PUT request to the comment you created, follow steps above for the posts but for the comments resource
